functional_tests/.coveragerc

```diff
@@ -4,3 +4,11 @@ parallel = True
 # TODO(jhr): enable this in the future
 # branch = True
 concurrency = multiprocessing,thread
+
+omit =
+    metaflow/*
+
+[report]
+
+omit =
+    metaflow/*
```

Comment:
Why are we omitting metaflow from test coverage?

Reply:
`func-lin-py37` was failing because of a coverage issue even though functional tests were passing.  Trying to resolve this now.

Reply:
Fix/workaround for all the coverage issues is at:
https://github.com/wandb/client/pull/2563